### PR TITLE
Set Selenium timeouts explicitly

### PR DESCRIPTION
**Pull request by tinez**
Signed-off-by: Marcin Sobczyk <msobczyk@redhat.com>

### CODE REVIEW

**Comment by michalskrivanek** [comment]
yeah...so i woud add a sleep as well

File: basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py [hunk 1]
```diff
@@ -234,6 +236,7 @@ def test_non_admin_login_to_webadmin(
     assert welcome_screen.is_user_logged_in(nonadmin_username)
     welcome_screen.logout()
     assert welcome_screen.is_user_logged_out()
+    time.sleep(5)
```
Comment (ljelinkova) [inline basic-suite-master/test-scenarios/test_100_basic_ui_sanity.py:239]: Why?

**Comment by michalskrivanek** [comment]
and i think we need https://github.com/oVirt/ovirt-engine/pull/704 too

**Comment by michalskrivanek** [comment]
> > and i think we need [oVirt/ovirt-engine#704](https://github.com/oVirt/ovirt-engine/pull/704) too
> 
> https://redir.apps.ovirt.org/dj/job/ds-ost-baremetal_manual/53956

https://redir.apps.ovirt.org/dj/job/ds-ost-baremetal_manual/53968

**Review by michalskrivanek — APPROVED** [review]

**Pull request merged**